suite.yml

```diff
@@ -66,13 +66,21 @@ section:
   - name: DevOps Tools
     description: Conjur OSS integrations with DevOps tools.
     repos:
+      - name: cyberark/ansible-conjur-collection
+        url: https://github.com/cyberark/ansible-conjur-collection
+        tool: Ansible
+        description: Conjur Ansible collection for use with Ansible 2.9+.
+          Includes an Ansible role to bootstrap Ansible hosts with a Conjur
+          identity, and a lookup plugin to enable easy access to Conjur secrets
+          from within your Ansible playbooks, etc.
+        version: v1.1.0
       - name: cyberark/ansible-conjur-host-identity
         url: https://github.com/cyberark/ansible-conjur-host-identity
         tool: Ansible
         description: Ansible role to provide Conjur machine identity to application
           hosts and install the Summon tool, which enables hosts to securely retrieve
           credentials.
-        version: v0.3.1
+        version: v0.3.2
```

Comment:
The first item under "Deprecated" in this release's CHANGELOG entry needs to be moved to "Removed".

Reply:
@alexkalish can you please approve: https://github.com/cyberark/ansible-conjur-host-identity/pull/47